osbs/build/spec.py

```diff
@@ -271,11 +272,21 @@ def set_params(self, git_uri=None, git_ref=None,
         self.yum_repourls.value = yum_repourls or []
         self.use_auth.value = use_auth
 
-        if build_imagestream and build_image:
+        unique_build_args = (build_imagestream, build_image, build_from)
+        if sum(bool(a) for a in unique_build_args) != 1:
```

Comment:
This might break existing configuration files which do not specify any of them and instead use the default value. Can we keep those working, perhaps by having a new default value "image:buildroot:latest" for build_from when no values are provided?